demo/templates/demo/includes/body.html

```diff
@@ -2,6 +2,6 @@
 
 {% if body %}
     <div class="body-content">
-      {{ body|richtext }}
+        {% include "demo/includes/streamfield.html" with content=body %}
```

Comment:
This breaks the rendering of pages where body is still a rich text field, such as StandardPage and PersonPage. (Personally I'd rather ditch the body.html include entirely - there's very little point in turning a trivial 3-line snippet into a reusable include.)